warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__schedule_feed_guideline_index.sql

```diff
@@ -0,0 +1,16 @@
+{{ config(materialized='ephemeral') }}
+
+WITH fct_daily_schedule_feeds AS (
+    SELECT * FROM {{ ref('fct_daily_schedule_feeds') }}
+),
+
+-- we never want results from the current date, as data will be incomplete
+int_gtfs_quality__schedule_feed_guideline_index AS (
+    SELECT
+        feed_key,
+        date,
+    FROM fct_daily_schedule_feeds
+    WHERE date < CURRENT_DATE
+)
+
+SELECT * FROM int_gtfs_quality__schedule_feed_guideline_index
```

Comment:
Is "schedule feed guideline index" an appropriate name for this? Since this table has a row per feed per day, it seems like perhaps "feed daily index" would be more accurate. This table doesn't seem to care about guidelines.

Reply:
The upstream model contains dates going into the future, whereas for guidelines we cannot check anything earlier than the current day, which I don't think is technically true for the data itself. Even if this model doesn't contain guidelines/checks, it's an index for those checks rather than for general data work. Is that reasonable?

Reply:
Yeah that's reasonable, thanks for explaining